src/middleware/src/storage/src/websql.js

```diff
@@ -7,9 +7,9 @@ import isFunction from 'lodash/isFunction';
 import isString from 'lodash/isString';
 const idAttribute = process.env.KINVEY_ID_ATTRIBUTE || '_id';
 const masterCollectionName = 'sqlite_master';
-const size = 5 * 1000 * 1000; // Database size in bytes
 let dbCache = {};
-let isSupported = undefined;
+let isSupported;
+const SIZE = 5000;
```

Comment:
@thomasconner Again, why `5000`? It was `5 * 1000 * 1000` before. Am I missing something?